tcms/settings/common.py

```diff
@@ -208,11 +208,12 @@
 ]
 
 # If you set this to False, Django will not use timezone-aware datetimes.
-USE_TZ = False
+USE_TZ = True if os.environ.get('KIWI_USE_TZ') == str(True) else False
```

Comment:
What is your intention here? Do you want to use timezone only when `$KIWI_USE_TZ` is set to the string "True" or any other non-empty string ? 

Depending on what behaviour you want you either need a comment to document that or use a simpler boolean condition (e.g. using `==` will return bool and you can also compare strings directly, can use `.lower()` for convenience).

Reply:
My intention is to use timezone ONLY when env  KIWI_USE_TZ is set to True.  non-empty string , e.g. bool('False'), will resolve to `true` in python. prefer not using `.lower()` to follow Django settings.

Reply:
comment has been updated. re-check.

Reply:
`os.environ.get('KIWI_USE_TZ', 'False').lower() == 'true'` evaluates as boolean and allows the user to specify any capitalization variant of "True". If you don't want to allow variations then `os.environ.get('KIWI_USE_TZ') == 'True'` is still simpler than the if expression you have currently.

Reply:
if KIWI_USE_TZ is set too False or any non-empty string, `os.envuse iron.get('KIWI_USE_TZ')`  will resolve to True, this is why I use `str(True)` in the code to make sure only "True" is taken as authentic.

Reply:
if KIWI_USE_TZ is set too False or any non-empty string, `os.envuse iron.get('KIWI_USE_TZ')`  will resolve to True, this is why I use `str(True)` in the code to make sure only "True" is taken as authentic.

Reply:
> if KIWI_USE_TZ is set to False or any non-empty string, os.envuse iron.get('KIWI_USE_TZ') will resolve to True

This is simply not True. `os.environ.get` returns a string, it isn't automatically resolved as boolean. This is why you compare the result with `==` to another string and the result of the equality comparison operator is boolean:

```
>>> use_tz = os.environ.get('KIWI_USE_TZ')
>>> use_tz
'False'
>>> type(use_tz)
<type 'str'>
>>> use_tz.lower() == 'true'
False
```

Reply:
Ok, updated.

Reply:
I don't see an update. This should become:
`USE_TZ = os.environ.get('KIWI_USE_TZ').lower() == 'true'`

Reply:
re-check.